js/apps/admin-ui/src/clients/utils.ts

```diff
@@ -4,7 +4,8 @@ import type { TFunction } from "i18next";
 /**
  * Checks if a client is intended to be used for authenticating a to a realm.
  */
-export const isRealmClient = (client: ClientRepresentation) => !client.protocol;
+export const isRealmClient = (client: ClientRepresentation) =>
+  client.attributes?.["realm_client"];
```

Comment:
What is the value of `realm_client`? If it is a string there will need to be an explicit comparison.
```suggestion
  client.attributes?.["realm_client"] === true.toString();
```

Reply:
Done, thanks!